src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -507,4 +508,7 @@ private static boolean isCurrentManager(Long activeFlag, LocalDate endDate) {
         return result;
     }
 
+    public static OffenderAssessments assessmentsOf(Offender offender) {
```

Comment:
There is an OffenderTransformer test class - consider adding a unit test on this method.

Reply:
Thanks for the feedback, I've done so